### PR TITLE
EN Form Generator Guide Cleanup

### PR DESCRIPTION
**Pull request by blairwinans**
My only question is how we are handling updating the links, as I left a lot of them intact but going to their /de/ counterparts. Let me know if there is anything I should do with those, along with any issues you may see. If I can get into a rhythm with these my hope is to help with a bunch. :) Thank you!

### CODE REVIEW

File: docs/manual/form-generator/_index.en.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
 ---
 title: 'Form Generator'
```
Comment (netzarbeiter) [inline docs/manual/form-generator/_index.en.md]:
```suggestion
title: "Form Generator"
```

File: docs/manual/form-generator/_index.en.md [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
-description: 'The Contao form generator supports you in creating forms by abstracting both the process of code generation and the validation of user input.'
+description: 'The Contao form generator helps you create forms by abstracting both the process of code generation and the complexity of validating the user input.'
```
Comment (netzarbeiter) [inline docs/manual/form-generator/_index.en.md]:
```suggestion
description: "The Contao form generator helps you create forms by abstracting both the process of code generation and the complexity of validating the user input."
```

File: docs/manual/form-generator/_index.en.md [hunk 3]
```diff
@@ -1,15 +1,13 @@
...
 aliases:
     - /en/form-generator/_index/
```
Comment (netzarbeiter) [inline docs/manual/form-generator/_index.en.md]:
```suggestion
    - /en/form-generator/
```

File: docs/manual/form-generator/create-a-search-form.en.md [hunk 4]
```diff
@@ -1,20 +1,23 @@
 ---
 title: 'Create a search form'
```
Comment (netzarbeiter) [inline docs/manual/form-generator/create-a-search-form.en.md]:
```suggestion
title: "Create a search form"
```

File: docs/manual/form-generator/create-a-search-form.en.md [hunk 5]
```diff
@@ -1,20 +1,23 @@
...
-description: 'You can use the form generator to create your own search form and include it in the header of your website, for example.'
+description: 'You can use the form generator to create your own search form and include it in the header of your website, as an example.'
```
Comment (netzarbeiter) [inline docs/manual/form-generator/create-a-search-form.en.md]:
```suggestion
description: "You can use the form generator to create your own search form and include it in the header of your website, as an example."
```

File: docs/manual/form-generator/create-a-search-form.en.md [hunk 6]
```diff
@@ -1,20 +1,23 @@
...
 aliases:
     - /en/form-generator/create-a-search-form/
 weight: 30
 ---
 
-{{% notice warning %}}
-This article is machine translated.
-{{% /notice %}}
+As already mentioned in the section [My Search Forms](../../modulverwaltung/website-suche/#eigene-suchformulare), you
```
Comment (netzarbeiter) [inline docs/manual/form-generator/create-a-search-form.en.md]:
```suggestion
As already mentioned in the section [My Search Forms](../../module-management/website-search/#own-search-forms), you 
```

File: docs/manual/form-generator/create-a-search-form.en.md [hunk 7]
```diff
@@ -1,20 +1,23 @@
...
+You only need to perform the following steps:
 
-As already mentioned in the section [My Search Forms](../../modulverwaltung/website-suche/#eigene-suchformulare), you can use the form generator to create your own search form and include it in the header of your website, for example. The following steps are necessary:
-
-1. Open the form generator, and create[ a new form](../../formulargenerator/formulare/#formular-konfiguration). Specify the search page as the redirect destination, and select *GET* as the transfer method.
-2. Add a [text field](../../formulargenerator/formularfelder/#textfeld) to the form, and enter the field `keywords`name. If you like, you can make this field mandatory, because a search without a search term will probably not yield many results.
-3. Optionally add a [radio button menu](../../formulargenerator/formularfelder/#radio-button-menue) to the form if you want your visitor to be able to choose between AND and OR search. The field name is in this case `query_type`and the option `and`values and `or`.
+1. Open the form generator, and create[ a new form](../../formulargenerator/formulare/#formular-konfiguration). Specify
```
Comment (netzarbeiter) [inline docs/manual/form-generator/create-a-search-form.en.md]:
```suggestion
1. Open the form generator, and create [a new form](../../form-generator/forms/#form-configuration). Specify 
```

File: docs/manual/form-generator/create-a-search-form.en.md [hunk 8]
```diff
@@ -1,20 +1,23 @@
...
-
-1. Open the form generator, and create[ a new form](../../formulargenerator/formulare/#formular-konfiguration). Specify the search page as the redirect destination, and select *GET* as the transfer method.
-2. Add a [text field](../../formulargenerator/formularfelder/#textfeld) to the form, and enter the field `keywords`name. If you like, you can make this field mandatory, because a search without a search term will probably not yield many results.
-3. Optionally add a [radio button menu](../../formulargenerator/formularfelder/#radio-button-menue) to the form if you want your visitor to be able to choose between AND and OR search. The field name is in this case `query_type`and the option `and`values and `or`.
+1. Open the form generator, and create[ a new form](../../formulargenerator/formulare/#formular-konfiguration). Specify 
+the search page as the redirect page, and select *GET* as the transfer method.
+2. Add a [text field](../../formulargenerator/formularfelder/#textfeld) to the form, and enter the field `keywords` name.
```
Comment (netzarbeiter) [inline docs/manual/form-generator/create-a-search-form.en.md]:
```suggestion
2. Add a [text field](../../form-generator/form-fields/#text-field) to the form, and enter the field `keywords` name. 
```

File: docs/manual/form-generator/create-a-search-form.en.md [hunk 9]
```diff
@@ -1,20 +1,23 @@
...
-2. Add a [text field](../../formulargenerator/formularfelder/#textfeld) to the form, and enter the field `keywords`name. If you like, you can make this field mandatory, because a search without a search term will probably not yield many results.
-3. Optionally add a [radio button menu](../../formulargenerator/formularfelder/#radio-button-menue) to the form if you want your visitor to be able to choose between AND and OR search. The field name is in this case `query_type`and the option `and`values and `or`.
+1. Open the form generator, and create[ a new form](../../formulargenerator/formulare/#formular-konfiguration). Specify 
+the search page as the redirect page, and select *GET* as the transfer method.
+2. Add a [text field](../../formulargenerator/formularfelder/#textfeld) to the form, and enter the field `keywords` name. 
+If you prefer, you can make this field mandatory, because a search without a search term will probably not yield many results.
+3. Optionally add a [radio button menu](../../formulargenerator/formularfelder/#radio-button-menue) to the form if you
```
Comment (netzarbeiter) [inline docs/manual/form-generator/create-a-search-form.en.md]:
```suggestion
3. Optionally add a [radio button menu](../../form-generator/form-fields/#radio-button-menu) to the form if you 
```

File: docs/manual/form-generator/create-a-search-form.en.md [hunk 10]
```diff
@@ -1,20 +1,23 @@
...
-3. Optionally add a [radio button menu](../../formulargenerator/formularfelder/#radio-button-menue) to the form if you want your visitor to be able to choose between AND and OR search. The field name is in this case `query_type`and the option `and`values and `or`.
+1. Open the form generator, and create[ a new form](../../formulargenerator/formulare/#formular-konfiguration). Specify 
+the search page as the redirect page, and select *GET* as the transfer method.
+2. Add a [text field](../../formulargenerator/formularfelder/#textfeld) to the form, and enter the field `keywords` name. 
+If you prefer, you can make this field mandatory, because a search without a search term will probably not yield many results.
+3. Optionally add a [radio button menu](../../formulargenerator/formularfelder/#radio-button-menue) to the form if you 
+want your visitor to be able to choose between AND and OR search. The field name is in this case `query_type` with the 
+option values `and` and `or`.
 4. Add a [submit button](../../formulargenerator/formularfelder/#absendefeld) to the form.
```
Comment (netzarbeiter) [inline docs/manual/form-generator/create-a-search-form.en.md]:
```suggestion
4. Add a [submit button](../../form-generator/form-fields/#submit-field) to the form.
```

File: docs/manual/form-generator/form-fields.en.md [hunk 11]
```diff
@@ -1,37 +1,39 @@
 ---
 title: 'Form fields'
```
Comment (netzarbeiter) [inline docs/manual/form-generator/form-fields.en.md:2]:
```suggestion
title: "Form fields"
```

File: docs/manual/form-generator/form-fields.en.md [hunk 12]
```diff
@@ -1,37 +1,39 @@
...
-description: 'Similar to articles and content elements, forms have a separate element for each form field, which is specifically designed to meet the particular requirements of the input field.'
+description: 'Similar to articles and content elements, forms will contain a separate element for each form field, which
```
Comment (netzarbeiter) [inline docs/manual/form-generator/form-fields.en.md:3]:
```suggestion
description: "Similar to articles and content elements, forms will contain a separate element for each form field, which 
```

File: docs/manual/form-generator/form-fields.en.md [hunk 13]
```diff
@@ -1,37 +1,39 @@
 ---
 title: 'Form fields'
-description: 'Similar to articles and content elements, forms have a separate element for each form field, which is specifically designed to meet the particular requirements of the input field.'
+description: 'Similar to articles and content elements, forms will contain a separate element for each form field, which 
+is specifically configured to meet the requirements for that field.'
```
Comment (netzarbeiter) [inline docs/manual/form-generator/form-fields.en.md:4]:
```suggestion
is specifically configured to meet the requirements for that field."
```

File: docs/manual/form-generator/form-fields.en.md [hunk 14]
```diff
@@ -88,27 +92,29 @@ The form field Text Field adds a single-line input field to the form. You should
...
+**Default value:** Here you can enter a default value. For accessible web pages, it is recommended to use the @ character 
+for e-mail addresses.
 
 **Minimum input length:** Here you can specify the minimum number of characters that can be entered in the text field.
 
 **Maximum input length**: Here you can specify the maximum number of characters that can be entered in the text field.
 
-**Keyboard shortcut:** A keyboard shortcut allows a visitor to jump directly to a specific input field by pressing the `[Alt]`- or - `[Strg]`key in conjunction with the keyboard shortcut, e.g. a number[ (see backend keyboard shortcut](../../administrationsbereich/backend-tastaturkuerzel/)).
+**Keyboard shortcut**: A keyboard shortcut allows a visitor to jump directly to a certain input field by pressing the 
+`[Alt]` or `[Strg]` key in connection with the shortcut key, e.g. a number[ (see backend keyboard shortcuts](../../administrationsbereich/backend-tastaturkuerzel/)).
```
Comment (netzarbeiter) [inline docs/manual/form-generator/form-fields.en.md:110]:
```suggestion
`[Alt]` or `[Strg]` key in connection with the shortcut key, e.g. a number ([see backend keyboard shortcuts](../../administrationsbereich/keyboard-shortcuts/)).
```

File: docs/manual/form-generator/form-fields.en.md [hunk 15]
```diff
@@ -121,92 +127,100 @@ The form field Text Field adds a single-line input field to the form. You should
...
 
 **Default value**: Here you can enter a default value.
 
 **Minimum input length**: Here you can specify the minimum number of characters that can be entered in the text field.
 
 **Maximum input length**: Here you can specify the maximum number of characters that can be entered in the text field.
 
-**Keyboard shortcut:** A keyboard shortcut allows a visitor to jump directly to a specific input field by pressing the `[Alt]`- or - `[Strg]`key in conjunction with the keyboard shortcut, e.g. a number[ (see backend keyboard shortcut](../../administrationsbereich/backend-tastaturkuerzel/)).
+**Keyboard shortcut**: A keyboard shortcut allows a visitor to jump directly to a certain input field by pressing the 
+`[Alt]` or `[Strg]` key in connection with the shortcut key, e.g. a number[ (see backend keyboard shortcuts](../../administrationsbereich/backend-tastaturkuerzel/)).
```
Comment (netzarbeiter) [inline docs/manual/form-generator/form-fields.en.md:171]:
```suggestion
`[Alt]` or `[Strg]` key in connection with the shortcut key, e.g. a number ([see backend keyboard shortcuts](../../administrationsbereich/keyboard-shortcuts/)).
```

File: docs/manual/form-generator/form-fields.en.md [hunk 16]
```diff
@@ -121,92 +127,100 @@ The form field Text Field adds a single-line input field to the form. You should
...
+make a line a group heading, select the Group option.
 
-![JavaScript wizard for creating options](/de/form-generator/images/de/anlegen-von-optionen.png?classes=shadow)
+![JavaScript wizard for creating options](/en/form-generator/images/en/select-options-wizard.png?classes=shadow)
 
 **CSS class:** Here you can enter one or more classes.
 
-**Shortcut key:** A shortcut key allows a visitor to jump directly to a certain input field by pressing the `[Alt]``[Strg]`key in conjunction with the shortcut key, e.g. a number[ (see backend shortcut key](../../administrationsbereich/backend-tastaturkuerzel/)).
+**Keyboard shortcut**: A keyboard shortcut allows a visitor to jump directly to a certain input field by pressing the 
+`[Alt]` or `[Strg]` key in connection with the shortcut key, e.g. a number[ (see backend keyboard shortcuts](../../administrationsbereich/backend-tastaturkuerzel/)).
```
Comment (netzarbeiter) [inline docs/manual/form-generator/form-fields.en.md:211]:
```suggestion
`[Alt]` or `[Strg]` key in connection with the shortcut key, e.g. a number ([see backend keyboard shortcuts](../../administrationsbereich/keyboard-shortcuts/)).
```

File: docs/manual/form-generator/form-fields.en.md [hunk 17]
```diff
@@ -304,32 +320,39 @@ The form field `Checkbox-Menü`adds a list of options to the form, from which y
...
-**Preserve existing files:** By default, Contao replaces a file as soon as a newer file with the same name is uploaded. If you select this option, existing files are preserved and new files are given a numeric suffix if they have the same name.
+**Preserve existing files:** By default, Contao overwrites a file as soon as a newer file with the same name is uploaded. 
+If you select this option, existing files are preserved, and new files are given a numeric suffix if they have the same name.
 
-**CSS class:** Here you can enter one or more classes.
+**CSS class:** Here you can enter one or more CSS classes.
 
-**Shortcut** key: A shortcut key allows a visitor to jump directly to a certain input field by pressing the `[Alt]``[Strg]`key in connection with the shortcut key, e.g. a number[ (see backend shortcut key](../../administrationsbereich/backend-tastaturkuerzel/)).
+**Keyboard shortcut**: A keyboard shortcut allows a visitor to jump directly to a certain input field by pressing the 
+`[Alt]` or `[Strg]` key in connection with the shortcut key, e.g. a number[ (see backend keyboard shortcuts](../../administrationsbereich/backend-tastaturkuerzel/)).
```
Comment (netzarbeiter) [inline docs/manual/form-generator/form-fields.en.md:346]:
```suggestion
`[Alt]` or `[Strg]` key in connection with the shortcut key, e.g. a number ([see backend keyboard shortcuts](../../administrationsbereich/keyboard-shortcuts/)).
```

File: docs/manual/form-generator/form-fields.en.md [hunk 18]
```diff
@@ -363,46 +387,50 @@ Hidden fields have no CSS class.
...
+The form field generates the following HTML code:
 
 ```html
 <input type="hidden" name="…" value="…">
 ```
 
 ## Security question
 
-The form field `Sicherheitsfrage`adds a [CAPTCHA](https://de.wikipedia.org/wiki/Captcha) to the form.
+The form field `Security question` adds a [CAPTCHA](https://de.wikipedia.org/wiki/Captcha) to the form.
```
Comment (netzarbeiter) [inline docs/manual/form-generator/form-fields.en.md:401]:
```suggestion
The form field `Security question` adds a [CAPTCHA](https://en.wikipedia.org/wiki/CAPTCHA) to the form.
```

File: docs/manual/form-generator/form-fields.en.md [hunk 19]
```diff
@@ -363,46 +387,50 @@ Hidden fields have no CSS class.
...
+If it happens that a visitor is falsely identified as a spambot, they only have to solve a simple calculation. Submitted 
+form data will not be lost under any circumstances.
 
 **Placeholder:** This text is displayed as long as the field has not been filled in.
 
 **CSS class:** Here you can enter one or more classes.
 
-**Keyboard shortc**ut: With a keyboard shortcut a visitor can jump directly to a certain input field by pressing the `[Alt]`- or - `[Strg]`key in connection with the keyboard shortcut, e.g. a number[ (see backend keyboard shortcut](../../administrationsbereich/backend-tastaturkuerzel/)).
+**Keyboard shortcut**: A keyboard shortcut allows a visitor to jump directly to a certain input field by pressing the 
+`[Alt]` or `[Strg]` key in connection with the shortcut key, e.g. a number[ (see backend keyboard shortcuts](../../administrationsbereich/backend-tastaturkuerzel/)).
```
Comment (netzarbeiter) [inline docs/manual/form-generator/form-fields.en.md]:
```suggestion
`[Alt]` or `[Strg]` key in connection with the shortcut key, e.g. a number ([see backend keyboard shortcuts](../../administrationsbereich/keyboard-shortcuts/)).
```

File: docs/manual/form-generator/form-fields.en.md [hunk 20]
```diff
@@ -419,24 +447,26 @@ If it happens that a visitor is falsely identified as a spambot, he only has to
...
 
-**source file:** Here you select the image for the screen button.
+**Source file:** Here you select the image for the image button.
 
-**CSS class**: Here you can enter one or more classes.
+**CSS class**: Here you can enter one or more CSS classes.
 
-**Keyboard shortc**ut: With a keyboard shortcut a visitor can jump directly to a certain input field by pressing the `[Alt]`- or - `[Strg]`key in connection with the keyboard shortcut, e.g. a number[ (see backend keyboard shortcut](../../administrationsbereich/backend-tastaturkuerzel/)).
+**Keyboard shortcut**: A keyboard shortcut allows a visitor to jump directly to a certain input field by pressing the 
+`[Alt]` or `[Strg]` key in connection with the shortcut key, e.g. a number[ (see backend keyboard shortcuts](../../administrationsbereich/backend-tastaturkuerzel/)).
```
Comment (netzarbeiter) [inline docs/manual/form-generator/form-fields.en.md]:
```suggestion
`[Alt]` or `[Strg]` key in connection with the shortcut key, e.g. a number ([see backend keyboard shortcuts](../../administrationsbereich/keyboard-shortcuts/)).
```

File: docs/manual/form-generator/forms.en.md [hunk 21]
```diff
@@ -6,13 +6,10 @@ aliases:
...
-{{% /notice %}}
-
-With the form generator you can create forms and either send their data by e-mail or write them into the database. Contao automatically checks the form input according to the rules you specify. TransferFiles can be sent as attachments or saved on the server.
+With the form generator you can create forms and either send their data by e-mail or write them to the database. Contao 
+automatically validates the form input according to the rules you specify. Files can be sent as attachments or saved on the server.
 
-## Own SMTP server
+## Setting your own SMTP server
 
 Without specifying a separate SMTP server, the data is [sent](https://de.wikipedia.org/wiki/Sendmail) via [sendmail](https://de.wikipedia.org/wiki/Sendmail), which can lead to problems.
```
Comment (netzarbeiter) [inline docs/manual/form-generator/forms.en.md]:
```suggestion
Without specifying a separate SMTP server, the data is sent via [sendmail](https://en.wikipedia.org/wiki/Sendmail), which can lead to problems.
```

File: docs/manual/form-generator/forms.en.md [hunk 22]
```diff
@@ -6,13 +6,10 @@ aliases:
 weight: 10
```
Comment (netzarbeiter) [inline docs/manual/form-generator/forms.en.md:6]: title: "Forms"
description: "With the form generator you can create forms and either send their data by e-mail or write them into the database."

File: docs/manual/form-generator/form-fields.en.md [hunk 23]
```diff
@@ -88,27 +92,29 @@ The form field Text Field adds a single-line input field to the form. You should
...
-**Individual template:** Here you can overwrite the default template.
+**Individual template:** Here you can overwrite the default template with a custom one.
 
-**HTML outputThe**  
- form field generates the following HTML code:
+**HTML Output**  
+The form field generates the following HTML code:
 
 ```html
 <div class="widget widget-text … mandatory">
```
Comment (netzarbeiter) [inline docs/manual/form-generator/form-fields.en.md]: <span class="invisible">Mandatory field </span>…<span class="mandatory">*</span>

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Blair.

**Review by fritzmg — APPROVED** [review]
looks good to me 👍

**Review by netzarbeiter — APPROVED** [review]

**Comment by fritzmg** [comment]
@blairwinans do you need help with resolving the conflicts? Also could you may be enable "allow edits from maintainers" for this PR? It seems to be disabled.

**Comment by blairwinans** [comment]
@fritzmg Sorry! Had turned off my GitHub notifications. That is not an option I see, probably because I forked it into our company repo and not my personal one. Sorry! Should be better now? Next time I'll fork on my personal account.

**Comment by netzarbeiter** [comment]
Thank you very much, Blair.

**Pull request merged**